docker/compose/mozdef_base/Dockerfile

```diff
@@ -17,14 +17,9 @@ RUN \
                   make && \
   useradd -ms /bin/bash -d /opt/mozdef -m mozdef && \
   mkdir /opt/mozdef/envs && \
-  curl -s -L https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz  | tar -C /opt/mozdef/ -xz && \
-  cd /opt/mozdef/Python-$PYTHON_VERSION && \
-  ./configure && \
-  make && \
-  make install && \
-  rm -r /opt/mozdef/Python-$PYTHON_VERSION && \
   cd /opt/mozdef && \
   yum install -y mysql-devel \
+                    python \
```

Comment:
Is there any worry here that centos7 will update to python3, and thus we'd run into breaking changes? Can we make sure we install python27 via yum?

Also, we can delete the $PYTHON_VERSION environment variable at the top of this file.

Reply:
I'm not sure of a good way to require 2.7 via yum?

Reply:
CentOS packaging syntax isn't going to bump python to python2.7 unless CentOS 8.  This would violate the rules of RedHat.  I think we should leave this as is.